solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -0,0 +1,130 @@
+"""
+Provides preprocessing steps to be performed on the timeseries data.
+"""
+
+
+def preprocess(preprocess_func):
+    """Preprocessing decorator."""
```

Comment:
what are you thinking of putting in this function?

Reply:
Would interval resampling and alignment go here?

Reply:
Yes, eventually. I suppose it doesn't need to be a decorator but I was envisioning it taking care of the consistent preprocessing steps regardless of the fillin method to be used.

Reply:
Yeah, I think it would be useful as a stand alone function.